webdriver/tests/release_actions/META.yml

```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1128104
```

Comment:
This bug is about why the test was disabled in Chromium. I guess that might be relevant, so even though it doesn't say why it fails in wptrunner, let's link it.